apps/chrome-devtools/package.json

```diff
@@ -94,6 +95,7 @@
     "@o3r/components": "workspace:^",
     "@o3r/configuration": "workspace:^",
     "@o3r/core": "workspace:^",
+    "@o3r/design": "workspace:^",
```

Comment:
not needed

Reply:
True I will fix it in another PR as this one is merged :)

Reply:
https://github.com/AmadeusITGroup/otter/pull/1689